<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="fgdc_timeperd">
     <property name="title">
      <string>Time Period Information</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>What is the time period of the information represented in the data set?</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="fgdc_timeinfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="timeinfoPage1">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date (YYYYMMDD)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit0">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="timeinfoPage2">
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>BeginningDate (YYYYMMDD)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>End Date (YYYYMMDD)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="dateEdit_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="dateEdit_3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="timeinfoPage3">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Date</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Series of Dates (YYYYMMDD)</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="dateEdit_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Enter information for only one option, either 'Single Date', 'Date Range', or 'Multiple Dates'.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="fgdc_current">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ground condition</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>observed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>publication date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>See Supplemental Info</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Currentness Reference</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Single Date</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Range</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Multiple Dates</string>
       </property>
      </widget>
     </widget>
     <zorder>label_2</zorder>
     <zorder>fgdc_timeinfo</zorder>
     <zorder>label_3</zorder>
     <zorder>fgdc_current</zorder>
     <zorder>label_8</zorder>
     <zorder>groupBox</zorder>
     <zorder>dateEdit</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
